client/components/token-field/index.jsx

```diff
@@ -183,6 +183,12 @@ var TokenField = React.createClass( {
 			} );
 		}
 
+		if ( autoCapitalize ) {
+			Object.assign( props, {
+				autoCapitalize: autoCapitalize
+			} );
```

Comment:
Ah right, nice eye. The equivalent is actually the following if we don't want to mutate props.
```
Object.assign( {}, props, { autoCapitalize: autoCapitalize } );
```